tests/foreman/ui_airgun/test_remoteexecution.py

```diff
@@ -0,0 +1,242 @@
+        2. Navigate to the hosts page and select at least two hosts
+        3. Click the "Select Action"
+        4. Select the job and appropriate template
+        5. Run the job
+
+    :expectedresults: Verify the job was successfully ran against the hosts
+
+    :CaseLevel: System
+    """
+    with VirtualMachine(distro=DISTRO_DEFAULT) as client_1, VirtualMachine(
+            distro=DISTRO_DEFAULT) as client_2:
+        vm_clients = [client_1, client_2]
+        host_names = [client.hostname for client in vm_clients]
+        for client in vm_clients:
+            _setup_vm_client_host(client, module_org.label)
+        with session:
+            hosts = session.host.search(
+                ' or '.join([f'name="{hostname}"' for hostname in host_names]))
+            assert set(host['Name'] for host in hosts) == set(host_names)
+            job_status = session.host.schedule_remote_job(
+                host_names,
+                {
+                    'job_category': 'Commands',
+                    'job_template': 'Run Command - SSH Default',
+                    'template_content.command': 'ls',
+                    'schedule': 'Execute now',
+                }
+            )
+            assert job_status['overview']['job_status'] == 'Success'
+            assert (set(host_job['Host'] for host_job in job_status['overview']['hosts_table'])
+                    == set(host_names))
+            assert all(host_job['Status'] == 'success'
+                       for host_job in job_status['overview']['hosts_table'])
+
+
+@tier3
+def test_positive_run_scheduled_job_template_by_ip(session, module_vm_client_by_ip):
+    """Schedule a job to be ran against a host by ip
+
+    :id: 4387bed9-969d-45fb-80c2-b0905bb7f1bd
+
+    :Setup: Use pre-defined job template.
+
+    :Steps:
+
+        1. Set remote_execution_connect_by_ip on host to true
+        2. Navigate to an individual host and click Run Job
+        3. Select the job and appropriate template
+        4. Select "Schedule future execution"
+        5. Enter a desired time for the job to run
+        6. Click submit
+
+    :expectedresults:
+
+        1. Verify the job was not immediately ran
+        2. Verify the job was successfully ran after the designated time
+
+    :CaseLevel: System
+    """
+    job_time = 180
+    hostname = module_vm_client_by_ip.hostname
+    with session:
+        plan_time = session.browser.get_client_datetime() + datetime.timedelta(seconds=job_time)
+        assert session.host.search(hostname)[0]['Name'] == hostname
+        job_status = session.host.schedule_remote_job(
+            [hostname],
+            {
+                'job_category': 'Commands',
+                'job_template': 'Run Command - SSH Default',
+                'template_content.command': 'ls',
+                'schedule': 'Schedule future execution',
+                'schedule_content.start_at': plan_time.strftime("%Y-%m-%d %H:%M"),
+            },
+            timeout=job_time
```

Comment:
that is a nice functionality, but it will be good if you explicitly verify on test layer that job is in queue state all the time, till it switch to success after 180 seconds. I feel that it should not be hidden in framework, but be present here